### PR TITLE
Updates the profiles on the civic tech jobs page

### LINKED ISSUE #6638: Update Project Profile: Civic Tech Jobs (Remove Jen Chung, Melinda Sukosd, & Jenn Wu; Update Lu Feng)
### Prerequisites
1. Be a member of Hack for LA. (There are no fees to join.) If you have not joined yet, please follow the steps on our [Getting Started](https://www.hackforla.org/getting-started) page and attend an onboarding session.
2. You have already read our [How to Contribute to Hack for LA Guide](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md).

### Overview
We need to keep project information up to date so that visitors to the website can find accurate information.

### Action Items
- [x] In your IDE, open the `_projects/civic-tech-jobs.md` file
- [x] Observe the existing syntax of the front matter block [^1] in the file
- [x] Find the **leadership** variable and do the following:
  - [x] Remove
    ```
      - name: Jen Chung 
        github-handle: 
        role: UX/UI Design Lead
        links:
          slack: https://hackforla.slack.com/team/U02A6H5PVAA
          github: https://github.com/jenchuu
        picture: https://avatars.githubusercontent.com/jenchuu
    ```
  - [x] Remove
    ```
      - name: Melinda Sukosd
        role: UX Research Lead
        links:
          slack: https://hackforla.slack.com/team/U03T1G9F46P
          github: https://github.com/melkosm
        picture: https://avatars.githubusercontent.com/melkosm
    ```
  - [x] Remove
    ```
      - name: Jenn Wu
        github-handle:
        role: UX Researcher Lead
        links:
          slack: https://hackforla.slack.com/team/U05JS3V38TV
          github: https://github.com/jayywu
        picture: https://avatars.githubusercontent.com/jayywu 
    ```
  - [x] Find Lu Feng's profile:
    ```
      - name: Lu Feng
        github-handle:
        role: UX/UI Designer
        links:
          slack: https://hackforla.slack.com/team/U03NV47TG4X
          github: https://github.com/fenglugithub
        picture: https://avatars.githubusercontent.com/fenglugithub
    ```
    - [x] Move Lu's profile so it is after the other Product Managers' and Leads' profiles
    - [x] Update the profile by adding the GitHub handle `fenglugithub` to the **github-handle** variable and change the **role** to `UI/UX Lead` so the result is
      ```
        - name: Lu Feng
          github-handle: fenglugithub
          role: UI/UX Lead
          links:
            slack: https://hackforla.slack.com/team/U03NV47TG4X
            github: https://github.com/fenglugithub
          picture: https://avatars.githubusercontent.com/fenglugithub
      ```
- [x] Verify the changes by viewing the following in your local environment and include before and after screenshots with your pull request:
  - [x] Civic Tech Jobs page [^2]
- [x] Once your pull request is merged, go to #6583 and
  - [x] Check off this issue under the _Dependency_ section
  - [x] If all the issues in the _Dependency_ section are checked off, move #6583 to **Questions / In Review** column and uncheck the **Dependency label**.
    - <details>
        <summary>Click here to see how to uncheck the <b>Dependency label</b></summary>
        
      </details>

### Resources/Instructions
- This issue was initiated by #6583
[^1]: [Info about the front matter block](https://jekyllrb.com/docs/front-matter/)
[^2]: Project detailed info page URL: https://www.hackforla.org/projects/civic-tech-jobs

### PR DESCRIPTION
**Pull request by jonnyoe**
Fixes #6638 

### What changes did you make?
  - Removed the profiles of Jen Chung, Melinda Sukosd, and Jenn Wu
  - Moved and updated the profile for Lu Feng

### Why did you make the changes (we will use this info to test)?
  - Keeps Civic Tech Job project information up to date so that visitors to the website can find accurate information.

### Screenshots of Proposed Changes Of The Website  (if any, please do not screen shot code changes)

<details>
<summary>Visuals before changes are applied</summary>

![image]Screenshot 2024-04-23 at 10 20 51 PM

</details>

<details>
<summary>Visuals after changes are applied</summary>
  
![image]Screenshot 2024-04-23 at 10 21 09 PM

</details>

### CODE REVIEW

File: _projects/civic-tech-jobs.md [hunk 1]
```diff
@@ -103,6 +76,13 @@ leadership:
       slack: https://hackforla.slack.com/team/U05JS3BS9FD
       github: https://github.com/ngadoq
     picture: https://avatars.githubusercontent.com/ngadoq 
+  - name: Lu Feng
+    github-handle: fenglugithub
```
Comment (freaky4wrld) [inline _projects/civic-tech-jobs.md:80]: @jono4727 we don't want it to be 

```
- name: Lu Feng
    github-handle: fenglugithub

```

however it should follow the template provided in the issue

```
- name: Lu Feng
    github-handle: 

```

Comment (jonnyoe) [reply _projects/civic-tech-jobs.md:80]: It says in the issue to "Update the profile by adding the GitHub handle fenglugithub to the github-handle variable and change the role to UI/UX Lead". It's a little further down maybe you didn't see?

**Comment by Jperparas** [comment]
**availability:** 4/25
**ETA:** EoD 4/25

**Review by Jperparas — APPROVED** [review]
Great work on the issue!
 branch is labeled correctly and going into the correct branch, issue linked, correct code changes, made.  Visual changes are documented and consistent with  issue.

Keep it up!

File: _projects/civic-tech-jobs.md (hunk 1 above)
Comment (jphamtv) [reply _projects/civic-tech-jobs.md:80]: Hi @jono4727, you're correct. I'll mark this as resolved.

**Review by jphamtv — CHANGES_REQUESTED** [review]
Hi @jono4727, 

Please move Lu Feng's profile up so it's directly after any managers or leads of the project per the issue's instructions. Also, your branch has conflicts that need to be resolved. 

Make sure to update your screenshots after making the changes. Once the updates are complete, re-request reviews by clicking on the circular arrows icon next to the reviewers. 

Thanks for your work on this!

**Comment by jonnyoe** [comment]
Hey @jphamtv,

Okay yeah I see that now. I'll create a new pull request later tonight. Thank you for catching this.

**Pull request closed without merging**